recipes/photutils/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "photutils" %}
+{% set version = "0.3.2" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "a02a0b205c058467f10d1605acfcef8b56695a81005ee2136c64914e55c1b42d" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --offline --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy x.x
```

Comment:
@ocefpaf said:

> Do you want to try conda-forge/conda-forge.github.io#415 here @mwcraig?

Sure; we've been shipping unpinned versions of this for about 1.5 years with no complaints.